app.py

```diff
@@ -10,12 +10,13 @@
 from datetime import datetime
 import json
 import hashlib
+import signal
 import manage_docker
 import manage_rancher
 from apscheduler.schedulers.background import BackgroundScheduler
```

Comment:
Could we remove importing apscheduler (and code later that sets up the scheduler)?